.github/scripts/gen-build-failure-report.sh

```diff
@@ -46,6 +50,32 @@ touch "$BUILD_DIR/build-failure"
   echo '</details>'
   echo ''
 
+  for hs_err in $hs_err_files; do
+    echo "<details><summary><b>View HotSpot error log: "$hs_err"</b></summary>"
+    echo ''
+    echo '```'
+    echo "$hs_err:"
+    echo ''
+    cat "$hs_err"
+    echo '```'
+    echo '</details>'
+    echo ''
+  done
+
   echo ''
   echo ':arrow_right: To see the entire test log, click the job in the list to the left. To download logs, see the `failure-logs` [artifact above](#artifacts).'
 ) >> $GITHUB_STEP_SUMMARY
+
+# With large hs_errs, the summary can easily exceed 1024 kB, the limit set by Github
```

Comment:
This code duplication is nagging me a bit. Do you think it would make sense to extract it to a separate helper in `.github/scripts`?

Reply:
Right. Let's see if I can do this without breaking much stuff.

Reply:
I think the separate script works!